inspirehep/config.py

```diff
@@ -889,3 +894,5 @@ def _(x):
     'stat.OT': 'Other',
     'stat.TH': 'Other'
 }
+
+ASSETS_DEBUG = True
```

Comment:
Well in theory it should slow loading times ;) as each JS creates a request to fetch the file VS production style bundles.